app/index.html

```diff
@@ -13,6 +13,7 @@
     <link rel="stylesheet" href="assets/cargo.css">
     <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
     <link rel="icon" href="/favicon.ico" type="image/x-icon">
+    <link rel="search" href="opensearch.xml" type="application/opensearchdescription+xml" title="Cargo">
```

Comment:
Should this be `/opensearch.xml` instead of `opensearch.xml` because this page is loaded from any path?